zerver/webhooks/mention/doc.md

```diff
@@ -1,27 +1,38 @@
+# Zulip Mention integration
+
 Get Mention notifications within Zulip via Zapier!
 
+{start_tabs}
+
 1. {!create-channel.md!}
 
 1. {!create-an-incoming-webhook.md!}
 
-1. {!generate-integration-url.md!}
+1. {!generate-webhook-url-basic.md!}
 
 1. Go to your Mention feed, and click on your profile in the top-right
-   corner. Select **Settings**. Click on **Integrations**. Select the
-   **Zapier** integration.
+   corner. Select **Settings**, click on **Integrations**, and Select
+   the **Zapier** integration.
 
 1. Click **Explore Mention on Zapier**. Search for "webhooks" in
-   the search bar, and click on **Webhooks by Zapier**. Scroll down
-   and look for **Add Webhook posts for new Mentions**, and click on
+   the search bar, and click on **Webhooks by Zapier**. Look for
+   **Add Webhook posts for new Mentions**, and click on
    **Use this Zap**. Click on **Create this Zap**.
 
-1. Follow the on-screen steps to link your
-   Mention account to Zapier and select your Mention **Account ID** and
-   **Alert** when prompted. Follow the on-screen steps to set up **Webhooks
-   by Zapier POST**. When prompted, set **URL** to the URL constructed
-   above, and set **Payload Type** to **JSON**. After **Test this Step**,
-   click **Finish**.
+1. Follow the on-screen steps to link your Mention account to Zapier,
+   select your Mention **Account ID**, and **Alert** when prompted.
```

Comment:
Let's make this a separate sentence: "Select your Mention **Account ID** and **Alert** when prompted."